resources/views/bread/edit-add.blade.php

```diff
@@ -116,8 +138,9 @@ class="form-edit-add"
 
                 <div class="modal-footer">
                     <button type="button" class="btn btn-default" data-dismiss="modal">{{ __('voyager::generic.cancel') }}</button>
-                    <button type="button" class="btn btn-danger" id="confirm_delete">{{ __('voyager::generic.delete_confirm') }}
-                    </button>
+                    @can('delete', $dataTypeContent)
```

Comment:
What's the point of this permissions check? You're already in the modal.  Might as well put the check around the entire modal at that point.

Reply:
Well spotted! I made a mistake here, wanted to put the permissions check around the delete button down in the footer. So after the update for this pull request it doesn't even show the delete button if you don't have permission to delete the record.